src/test/resources/json-merge-test/json-merge-test19.json

```diff
@@ -0,0 +1,5 @@
+[
+  {},
+  {},
+  {}
+]
```

Comment:
this test case is already written in https://github.com/hemantsonu20/json-merge/blob/master/src/test/resources/json-merge-test/edge-cases/edge-case1.json